stand/libsa/in_cksum.c

```diff
@@ -77,7 +77,7 @@ in_cksum(void *p, int len)
 #endif
 			}
 		}
-		if ((oddbyte = len & 1) != 0)
+		if ((oddbyte = len & 1) == 1)
```

Comment:
This isn't a workaround either...  This isn't a semantic change, per se, but why make it?

Reply:
To make the intention more explicit

Reply:
Comparing to zero is usually faster than comparing to any other value, because of the CPU flags that are computed after certain arithmetic instructions, like the result of binary AND, OR and XOR.

Reply:
The optimizer turns both of these options into the same object code

Reply:
Yes, exactly, so in these simple cases, maybe just write the code optimized, and save a few compiler cycles, who knows :-)